src/AppM.purs

```diff
@@ -33,8 +40,17 @@ derive newtype instance monadEffectAppM :: MonadEffect AppM
 
 derive newtype instance monadAffAppM :: MonadAff AppM
 
+derive newtype instance monadLoggerAppM :: MonadLogger AppM
+
 instance monadAskAppM :: TypeEquals o Options => MonadAsk o AppM where
   ask = AppM $ asks from
 
 runAppM :: forall a. AppM a -> Options -> Aff a
-runAppM (AppM m) options = runReaderT m options
+runAppM (AppM m) options = runReaderT (runLoggerT m (filterByVerbosity logMessage)) options
+  where
+  filterByVerbosity = case options.verbosity of
+    Verbosity.Info -> minimumLevel Info
+    Verbosity.Debug -> minimumLevel Debug
+    Verbosity.Silent -> const $ const $ pure unit
```

Comment:
Really nice feature! I had to shut down the console.log because tailwind css doesn't provide any slient/quiet mode, it's really useful though :(